### PR TITLE
feat!: upgrade http to 1.1.0

### LINKED ISSUE #979: dependant on old http version
**Describe the bug**
The http module version is now 1.0.0 starts to block other upgrades.

Please upgrade deps on HTTP module

### PR DESCRIPTION
**Pull request by jainkrati**
## Changes

In order to support solana on latest flutter apps which support http 1.1.0, I updated http version in pubspec.yaml

## Related issues

Fixes #979 

## Checklist

- [ ] PR is ready for review (if not, it should be a draft).
- [ ] PR title follows [Conventional Commits][1] guidelines.
- [ ] Screenshots/video added.
- [ ] Tests added.
- [ ] Self-review done.

### CODE REVIEW

File: packages/jsonrpc_client/pubspec.yaml [hunk 1]
```diff
@@ -9,12 +9,12 @@ environment:
...
   build: ^2.1.1
-  http: ^0.13.4
+  http: ^1.1.0
   solana:
     path: ../solana
   source_gen: ^1.1.1
 
 dev_dependencies:
-  dart_code_metrics: ^5.7.3
+  # dart_code_metrics: ^5.7.3
```
Comment (ookami-kb) [inline packages/jsonrpc_client/pubspec.yaml:18]: It shouldn't be commented.

File: packages/solana/pubspec.yaml [hunk 2]
```diff
@@ -17,15 +17,15 @@ dependencies:
...
+  http: ^1.1.0
   json_annotation: ^4.4.0
   typed_data: ^1.3.0
   web_socket_channel: ^2.1.0
 
 dev_dependencies:
   borsh: ^0.3.1+2
   build_runner: ^2.1.1
-  dart_code_metrics: ^5.7.3
+  # dart_code_metrics: ^5.7.3
```
Comment (ookami-kb) [inline packages/solana/pubspec.yaml:28]: Same here.

**Review by ookami-kb — CHANGES_REQUESTED** [review]

File: packages/jsonrpc_client/pubspec.yaml (hunk 1 above)
Comment (jainkrati) [reply packages/jsonrpc_client/pubspec.yaml:18]: Does dart_code_metrics support http 1.1.0 ?

Comment (jainkrati) [reply packages/jsonrpc_client/pubspec.yaml:18]: I was hoping if I could point to a version of dart_code_metrics which supports http 1.1.0, if it doesnt then we will need to upgrade its dependency as well but I cannot find the repo for dart_code_metrics

Comment (jainkrati) [reply packages/jsonrpc_client/pubspec.yaml]: Awesome. I have synced the latest code.

**Pull request closed without merging**